internal/bminventory/inventory.go

```diff
@@ -6199,6 +6199,11 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 		c = &cluster.Cluster
 	}
 
+	//day2 host is always a worker
+	if hostutil.IsDay2Host(host) {
```

Comment:
per my earlier comments - why is this in an if statement? it will always be true.

Reply:
Actually, it won't always be true. disregard this comment.